PayPalNative/src/main/java/com/braintreepayments/api/PayPalNativeClient.java

```diff
@@ -38,6 +48,17 @@ public void onResult(Configuration configuration, Exception error) {
         });
     }
 
+    private void sendVaultRequest(final FragmentActivity activity, final PayPalNativeVaultRequest payPalVaultRequest, final PayPalNativeFlowStartedCallback callback) {
+        //this one should default to the one we already have, once PayPalNative supports billing agreements, this should just default to native.
+        payPalClient = new PayPalClient(braintreeClient);
+        payPalClient.tokenizePayPalAccount(activity, payPalVaultRequest, new PayPalFlowStartedCallback() {
+            @Override
+            public void onResult(@Nullable Exception error) {
+                callback.onResult(error);
+            }
+        });
+    }
+
```

Comment:
In order to default to the same vault experience we have now, I just call the PayPalClient and wrap the callbacks.